packages/package_test.go

```diff
@@ -47,6 +47,182 @@ func TestLoadImageLabelDecodesYamlList(t *testing.T) {
+			errored := false
+			LintImage(
+				types.ImageInspect{
+					ContainerConfig: &container.Config{
+						Labels: map[string]string{"io.whalebrew.config.missing_volumes": strategy},
+					},
+					Config: &container.Config{
+						Entrypoint: []string{"/entrypoint"},
+					},
+				},
+				func(e error) {
+					errored = true
+					t.Errorf("no error should be raised")
+				},
+			)
+			assert.False(t, errored)
+		})
+	}
+}
+
+func TestLintImageErrorsWhenEntrypointIsMissing(t *testing.T) {
+	errored := false
+	LintImage(
+		types.ImageInspect{
+			Config: &container.Config{},
+		},
+		func(e error) {
+			errored = true
+			assert.Equal(t, NoEntrypointError{}, e)
+		},
+	)
+	assert.True(t, errored)
+}
+
+func TestLintImageErrorsWhenLabelDoesNotExitsFromContainerConfig(t *testing.T) {
+	errored := false
+	LintImage(
+		types.ImageInspect{
+			ContainerConfig: &container.Config{
+				Labels: map[string]string{"io.whalebrew.some.key": "- some\n- other"},
+			},
+			Config: &container.Config{
+				Entrypoint: []string{"/entrypoint"},
+			},
+		},
+		func(e error) {
+			errored = true
+			assert.Equal(t, UnknownLabelError{"io.whalebrew.some.key"}, e)
+		},
+	)
+	assert.True(t, errored)
+}
+
+func TestLintImageErrorsWhenLabelDoesNotExitsFromImageConfig(t *testing.T) {
+	errored := false
+	LintImage(
+		types.ImageInspect{
+			Config: &container.Config{
+				Labels:     map[string]string{"io.whalebrew.some.key": "- some\n- other"},
+				Entrypoint: []string{"/entrypoint"},
+			},
+		},
+		func(e error) {
+			errored = true
+			assert.Equal(t, UnknownLabelError{"io.whalebrew.some.key"}, e)
+		},
+	)
+	assert.True(t, errored)
+}
+
+func TestLintImageErrorsWhenLabelCantBeDecodedFromContainerConfig(t *testing.T) {
+	errored := false
+	LintImage(
+		types.ImageInspect{
+			ContainerConfig: &container.Config{
+				Labels: map[string]string{"io.whalebrew.config.environment": "content of label"},
+			},
+			Config: &container.Config{
+				Entrypoint: []string{"/entrypoint"},
+			},
+		},
+		func(e error) {
+			errored = true
+			assert.IsType(t, LabelError{}, e)
+			assert.Contains(t, e.Error(), "io.whalebrew.config.environment")
+			assert.Contains(t, e.Error(), "cannot unmarshal")
+			assert.Contains(t, e.Error(), "content of label")
+		},
+	)
+	assert.True(t, errored)
+}
+
+func TestLintImageErrorsWhenLabelCantBeDecodedFromImageConfig(t *testing.T) {
+	errored := false
+	LintImage(
+		types.ImageInspect{
+			Config: &container.Config{
+				Labels:     map[string]string{"io.whalebrew.config.environment": "content of label"},
+				Entrypoint: []string{"/entrypoint"},
+			},
+		},
+		func(e error) {
+			errored = true
+			assert.IsType(t, LabelError{}, e)
+			assert.Contains(t, e.Error(), "io.whalebrew.config.environment")
+			assert.Contains(t, e.Error(), "cannot unmarshal")
+			assert.Contains(t, e.Error(), "content of label")
+		},
+	)
+	assert.True(t, errored)
+}
+
+func TestLintImageSucceedsFromContainerConfig(t *testing.T) {
```

Comment:
There's no assert in this test.

Reply:
this is true, because we expect no call to the [error function](https://github.com/whalebrew/whalebrew/pull/126/files#diff-197cff7b2928534d73747c745560d49ee8048bacaa5d75487699d7fa56a2b313R206)
If the function is called, the test is [reported as errored](https://github.com/whalebrew/whalebrew/pull/126/files#diff-197cff7b2928534d73747c745560d49ee8048bacaa5d75487699d7fa56a2b313R207) and fails